velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -147,13 +146,17 @@ class ParquetTpchTest : public testing::Test {
         params, addSplits, duckQuery, *duckDb_, sortingKeys);
   }
 
-  std::shared_ptr<DuckDbQueryRunner> duckDb_;
-  std::shared_ptr<TempDirectoryPath> tempDirectory_;
-  TpchQueryBuilder tpchBuilder_{dwio::common::FileFormat::PARQUET};
+  static std::shared_ptr<DuckDbQueryRunner> duckDb_;
+  static std::shared_ptr<TempDirectoryPath> tempDirectory_;
+  static std::shared_ptr<TpchQueryBuilder> tpchBuilder_;
 
   static constexpr char const* kTpchConnectorId{"test-tpch"};
 };
 
+std::shared_ptr<DuckDbQueryRunner> ParquetTpchTest::duckDb_ = nullptr;
+std::shared_ptr<TempDirectoryPath> ParquetTpchTest::tempDirectory_ = nullptr;
+std::shared_ptr<TpchQueryBuilder> ParquetTpchTest::tpchBuilder_ = nullptr;
```

Comment:
is this necessary? shared-ptr are always initialized with nullptr by default.

Reply:
probably not. I will cleanup in a follow-up PR.